test/kubeobjects/namespace/namespace.go

```diff
@@ -14,9 +14,9 @@ import (
 	"sigs.k8s.io/e2e-framework/pkg/envconf"
 )
 
-func Create(name string) env.Func {
+func Create(name string, labels map[string]string) env.Func {
```

Comment:
Instead of having a create function which second parameter is `nil` most of the time, I'd suggest either creating a second function `CreateWithLabels`, using a builder pattern for a namespace to pass to a client, or create the desired namespace struct directly and pass it to a client.

Reply:
namespace builder pattern used